### PR TITLE
Control set_pos -> set_position

### LINKED ISSUE #8005: Control set_pos -> set_position
Since Node2D set_pos() has changed to set_position() in 3.0 I think Control set_pos() should also be renamed to keep everything consistent.

 
image

image

https://github.com/godotengine/godot/blob/5dbf1809c6e3e905b94b8764e99491e608122261/core/math/math_2d.h#L207
https://github.com/godotengine/godot/blob/5dbf1809c6e3e905b94b8764e99491e608122261/scene/gui/control.cpp#L60

See also that Node2d has a set_rotation_in_degrees(value) and set_rotation(value)
Control has only set_rotation(radians).

### PR DESCRIPTION
**Pull request by sergey-push**
fixes #8005

### CODE REVIEW

**Review by ficoos — APPROVED** [review]

File: scene/gui/control.cpp [hunk 1]
```diff
@@ -2314,7 +2314,7 @@ void Control::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_anchor_and_margin", "margin", "anchor_mode", "offset"), &Control::set_anchor_and_margin);
 	ClassDB::bind_method(D_METHOD("set_begin", "pos"), &Control::set_begin);
 	ClassDB::bind_method(D_METHOD("set_end", "pos"), &Control::set_end);
-	ClassDB::bind_method(D_METHOD("set_pos", "pos"), &Control::set_pos);
+	ClassDB::bind_method(D_METHOD("set_position", "pos"), &Control::set_position);
```
Comment (akien-mga) [inline scene/gui/control.cpp]: The same should be done for `{s,g}et_global_pos`.

**Comment by sergey-push** [comment]
@akien-mga done

**Comment by sergey-push** [comment]
who should resolve the conflict, me or someone with write access?

**Comment by volzhs** [comment]
@sergey-push it's better you to resolve it. :smile:

**Comment by akien-mga** [comment]
Merged manually as 8589ca3.

**Pull request closed without merging**